src/components/pagination/index.md.njk

```diff
@@ -0,0 +1,118 @@
+---
+title: Pagination
+description: Help users navigate collections of numbered pages like search results
+section: Components
+aliases:
+backlog_issue_id: 77
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+Help users navigate a series of pages like search results or guides.
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true }) }}
+
+## When to use this component
+
+Use this component when you need to present content, such as guidance or search results, across multiple pages.
+
+You might do this when:
+- showing all the content on a single page makes the page take too long to load
+- users may benefit from bookmarking or linking to specific pages of the content
+- most users will only need the content on the first pages
+
+## When not to use this component
+
+Only break up content into pages if it improves the performance or usability of your service.
+
+Avoid using the 'infinite scroll' technique to automatically load content when the user approaches the bottom of the page. This causes problems for keyboard users.
+
+## How it works
+
+Add the pagination component after the content on each page that you have broken up.
```

Comment:
This is really succinct, but it took me a few reads to 'get' it. Because, I think, we mention the user's second action first and vice-versa.

Wonder could we put them in the sequence a user would do them in. So, something like:

> First, break the overall content into pages. Then add the pagination component after the content on each page.

Or something like:

> Once you've broken content into separate pages, add the pagination component after the content on each page.